.circleci/config.yml

```diff
@@ -18,6 +18,20 @@ jobs:
     steps:
       - checkout
       - run: ci/do_ci.sh test
+  check-format:
+    docker:
+      - image: *envoy-build-image
+    resource_class: xlarge
+    steps:
+      - checkout
+      - run: ci/do_ci.sh check_format
+  unit_tests_coverage:
+    docker:
+      - image: *envoy-build-image
+    resource_class: xlarge
+    steps:
+      - checkout
+      - run: ci/do_ci.sh coverage
```

Comment:
@mum4k I'm not sure whether we should add these tests in circleci now, or just skip and add them in AZP in the future.

Reply:
Adding them now makes sense, that will keep them passing over time.

CircleCI is fine for anything small-ish, it just doesn't have enough resources to reliably build and run performance tests with Nighthawk and Envoy.

Reply:
Please add these into the workflows below so we can verify whether they execute successfully.